### ISSUE TITLE
What is `bpworkers()` supposed to do?

### ISSUE THREAD

**Issue by lawremi** [Bioconductor/BiocParallel#113]
The documentation for `bpworkers()` is:
```
‘bpworkers(x)’, ‘bpworkers(x, ...)’: ‘integer(1)’ or ‘character()’.
           Gets the number or names of the back-end workers. The setter
           is supported for SnowParam and MulticoreParam only.
```

That is confusing, why would the function sometimes return the number and sometimes the names of the workers? But when I call `bpworkers(SerialParam())` it returns `list(FALSE)` which is neither of those things.

**Comment by mtmorgan** [comment]
probably you are looking for `bpnworkers()`.

`bpworkers()` answers the question 'who are the workers'. This is sometimes implicit -- `bpworkers(SnowParam(2))` just know that there are two workers spawned in the default way, whereas `bpworkers(SnowParam(c("localhost", "127.0.0.1")))` knows the ip address / name of the workers. `bpworkers(SerialParam())` seems to be broken and I'll investigate...

**Comment by Jiefei-Wang** [comment]
The problem of `bpworkers(SerialParam())` has been fixed in Bioc 3.14
```
> bpworkers(SerialParam())
[1] 1
```
